less/index.less

```diff
@@ -53,7 +53,238 @@ table {
     box-sizing: border-box;
 }
 
+@darker-gray: #333;
+@lighter-gray: #bbb;
```

Comment:
Good job using variables here. Don't forget you can use these for other things too.